app/models/project.rb

```diff
@@ -21,6 +21,7 @@ class Project < ActiveRecord::Base
     state :proposed, initial: true
     state :accepted
     state :rejected
+    state :pending
 
     event :accept do
       transitions from: :proposed, to: :accepted
```

Comment:
Since we have an additional state `pending` now, we need to also allow transitioning from `:pending` in addition to `:proposed`.